File: sdkwork-backend-sdk-kotlin/src/main/kotlin/com/sdkwork/backend/MimeType.kt
package com.sdkwork.backend

data class MimeType(
    val type: String? = null,
    val subtype: String? = null,
    val parameters: Map<String, String>? = null,
    val charset: String? = null,
    val concrete: Boolean? = null,
    val subtypeSuffix: String? = null,
    val wildcardType: Boolean? = null,
    val wildcardSubtype: Boolean? = null
)

File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/mime_type.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class MimeType:
    type: str = None
    subtype: str = None
    parameters: Dict[str, str] = None
    charset: str = None
    concrete: bool = None
    subtype_suffix: str = None
    wildcard_type: bool = None
    wildcard_subtype: bool = None
